### ISSUE TITLE
Permanent error: InvalidParameterException: Log groups starting with AWS/ are reserved for AWS.

### ISSUE THREAD

**Issue by fbdo** [aws-observability/aws-otel-collector#353]
Hi aws-otel-collector team,

After configuring and making my app talk to my collector, I started to see again and again the following message in the aws-otel-collector logs:
```
2021-02-10T16:40:30.731Z	ERROR	exporterhelper/queued_retry.go:204	Exporting failed. Try enabling retry_on_failure config option.	
{
    "component_kind": "exporter",
    "component_type": "awsemf",
    "component_name": "awsemf/application",
    "error": "Permanent error: InvalidParameterException: Log groups starting with AWS/ are reserved for AWS."
}
```

Here my sudecar definition (using AWS CDK):
```
taskDefinition.addContainer('aws-otel-collector', {
      image: ecs.ContainerImage.fromRegistry('amazon/aws-otel-collector'),
      memoryLimitMiB: 512,
      cpu: 256,
      command: ['--config=/etc/ecs/container-insights/otel-task-metrics-config.yaml'],
      logging
    });
```

And the execution role policy:
```
private static ExecutionRolePolicy = new iam.PolicyStatement({
    effect: iam.Effect.ALLOW,
    resources: ['*'],
    actions: [
        "ecr:GetAuthorizationToken",
        "ecr:BatchCheckLayerAvailability",
        "ecr:GetDownloadUrlForLayer",
        "ecr:BatchGetImage",
        "logs:CreateLogStream",
        "logs:PutLogEvents",
        "xray:PutTraceSegments",
        "xray:PutTelemetryRecords"
    ]
  });
```

AS the error is coming from the collector itself, I wonder if I misconfigured something.

Any advice is much appreciated.

**Comment by hossain-rayhan** [comment]
Its issue with the config file. In [line 82](https://github.com/aws-observability/aws-otel-collector/blob/main/config/ecs/container-insights/otel-task-metrics-config.yaml#L86), we should append `/` before the log group name and I believe this will resolve the issue. 

We have an open PR, #330 should resolve this issue.

**Comment by pingleig** [comment]
@mxiamxia @hossain-rayhan did #400 solve this?

**Comment by hossain-rayhan** [comment]
@pingleig , I believe it was fixed. We should not receive this error if we are using the latest config.

**Comment by mxiamxia** [comment]
The issue is fixed in v0.8.0 release. close the issue.
